guide/README.md

```diff
@@ -10,7 +10,9 @@ MESG Services have two responsibilities:
 - [Executing **tasks**](/guide/service/listen-for-tasks.md) that an application triggers
 - [Sending **events**](/guide/service/emit-an-event.md) to an application
 
-[Applications](/guide/application/) rely only on [services](/guide/service/) managed by the MESG Engine, so developers can focus on applications' business logic and nothing more.
+Applications built with MESG are called processes.
```

Comment:
What do you think about removing this line. Just not to mix application and processes terms and start stright with process deifnition?

Reply:
I found it a bit weird to talk about process when people might not know already what it is but I would be ok with that